frontend/src/Components/MessageList.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import Container from 'react-bootstrap/Container';
+import Message from './Message';
+
+function MessageList(props) {
+	if (props.messages === undefined) {
+		return null;
```

Comment:
style wise, they like to wrap null in parens:
https://stackoverflow.com/a/42083262

Reply:
my editor is giving me a hard time on this one.